### DISCUSSION TITLE
Helpers: Ability to reorder Dropdown options

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11238] [Other feature requests] [answered]
## The request

When creating a Dropdown (aka input_select) in the new Helpers UI, there isn't currently a way to rearrange the order of these items. 

In this quick demonstration screenshot, I would want "My Favorite Playlist" to be listed as the very first option above all others. 

![input_select](https://i.imgur.com/mv387rd.png)

Perhaps some kind of "Reorder" button and when you press that it would add up/down arrows or drag handles.

**Comment by zsarnett** [comment]
We should add Sortable Library here like the sidebar and the entities picker

**Comment by almighty059** [comment]
No update on this.?

**Comment by matthiasdebaat** [comment]
There has been some work in progress on this, but isn't fixed yet https://github.com/home-assistant/frontend/issues/10107#issue-1011937944

**Comment by B0ndo2** [comment]
Any traction on this ?. It's really annoying not to have this functionality specially when the drop down needs to be sorted

**Comment by nkuttler** [comment]
I ended up editing `~/.homeassistant/.storage/input_select` directly and restarting hass. I don't recommend anybody to try this, you'll need to produce valid json, but it works.

**Comment by karwosts** [comment, marked as answer]
This is implemented in 2023.9 

https://github.com/home-assistant/frontend/pull/17706
